info.yml

```diff
@@ -206,6 +206,17 @@ configuration:
         description: Controls whether new tasks can be created from the app.
         default_value: True
 
+    file_browser_tabs:
+        type: list
+        description: "A list of tab names that are visible in the main browser. Values
+                      can be any combination of the following tab names: 'All',
+                      'Working', and 'Publishes'. The "
+
+        allows_empty: False
+        values:
+          type: str
+        default_value: ['All', 'Working', 'Publishes']
```

Comment:
I'm assuming the order in the list determines the order the tabs are shown in the UI? If so, the first value specifies what the user sees by default when they open the app? If that's the case, can that be explained better in the description above?

Reply:
yep. see previous comment.